src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/BulkPrintService.java

```diff
@@ -199,7 +185,7 @@ private UUID printDocuments(CaseDetails caseDetails, BulkPrintDocument coverShee
 
     private void generateCoversheetForRespondentAndSendOrders(CaseDetails caseDetails, String authToken) {
```

Comment:
could we pull out any order specific stuff to orderDocumentService and call re-useable respondent method?

Reply:
agree it should be done but not sure this is the ticket for it, happy to do it in my next bulkprint ticket